external/kafka/src/test/scala/org/apache/spark/streaming/kafka/KafkaStreamSuite.scala

```diff
@@ -30,6 +30,9 @@ import org.apache.spark.{SparkConf, SparkFunSuite}
 import org.apache.spark.storage.StorageLevel
 import org.apache.spark.streaming.{Milliseconds, StreamingContext}
 
+import java.util.concurrent.ConcurrentHashMap
+import scala.collection.convert.decorateAsScala._
```

Comment:
@holdenk Thanks for your comments.  Yes, that's why I have decorateAsScala there.  

I am working on changing the code to use Java API for +=, put and getOrElseUpdate.  Do we also need concurrency guarantee for ++ and --?